tools/tesseratos/src/tesseratos/metrics_panel/plugin.cpp

```diff
@@ -0,0 +1,66 @@
+#include <imgui.h>
+
+#include <cubos/engine/imgui/plugin.hpp>
+#include <cubos/engine/imgui/serialization.hpp>
+
+#include <tesseratos/metrics_panel/plugin.hpp>
+
+using cubos::core::ecs::Read;
+using cubos::core::ecs::Write;
+using cubos::engine::Cubos;
+using cubos::engine::DeltaTime;
+
+using namespace tesseratos;
+
+struct Metrics
+{
+    float timeElapsed = 0.0;
+    float fps = 0.0;
+    int frameCounter = 0;
+    float secondTimer = 0.0;
+    // FPSGraph related
+    std::vector<float> fpsHistory;
+};
+
+static void panel(Read<DeltaTime> deltatime, Write<Metrics> metrics)
```

Comment:
```suggestion
static void panel(Read<DeltaTime> deltaTime, Write<Metrics> metrics)
```

Reply:
Also, make the name of the system more explicit (and make it end in "system"). something like `showMetricsSystem`